### ISSUE TITLE
Double-size Emoji

### ISSUE THREAD

**Issue by Vortec4800** [MessageKit/MessageKit#794]
Is it possible to have larger emoji like Slack or iMessage app? I've set my message type to emoji, and I have the background color of the message switched to be clear on those messages, but the font size is the same as a normal text message. It looked like there were some older issues and commits that imply that this feature exists, but I'm not sure how to activate it.

**Comment by SD10** [comment]
@Vortec4800 You need to sent the `messageFont` property on `emojiMessageSizeCalculator` to the font you'd like to use for emoji messages. We could make this double the default size used for the `textMessageSizeCalculator` by default? I'd accept a PR

https://github.com/MessageKit/MessageKit/blob/d58c6a8a898c491084967ca260b02e8bcd95d9e9/Sources/Layout/MessagesCollectionViewFlowLayout.swift#L129

**Comment by Vortec4800** [comment]
Thanks for this. It took me a bit to figure out how to actually get access to this, but I was able to make it work.

I ended up adding this to my view controller's view did load method:

```
let layout = self.messagesCollectionView.collectionViewLayout as? MessagesCollectionViewFlowLayout
layout?.emojiMessageSizeCalculator.messageLabelFont = UIFont.systemFont(ofSize: 34.0)
```

I can try to work this in as a PR if you guys want this to be the default.

**Comment by SD10** [comment]
@Vortec4800 Nice work 👍 This is the font we use for the `textMessageSizeCalculator`:
https://github.com/MessageKit/MessageKit/blob/d58c6a8a898c491084967ca260b02e8bcd95d9e9/Sources/Layout/TextMessageSizeCalculator.swift#L32

The emoji messages should have a default font that is 2x this size

**Comment by SD10** [comment]
Fixed in #795
